code/modules/mob/living/carbon/human/species/station/skrell/skrell.dm

```diff
@@ -4,6 +4,7 @@
 	name_plural = "Skrell"
 	bodytype = "Skrell"
 	age_max = 500
+	default_genders = list(PLURAL)
```

Comment:
This will not make all Skrell plural gender. See special spawners like /datum/ghostspawner/human/rescuepodsurv

You will need to make some changes in many places for this to affect all species in all circumstances.